packages/storybook-docs/src/components/Use.tsx

```diff
@@ -1,41 +1,35 @@
 import type { HTMLAttributes, ReactElement, PropsWithChildren } from 'react';
-// eslint-disable-next-line @talend/import-depth
-import { IconNameWithSize } from '@talend/icons/dist/typeUtils';
 import classnames from 'classnames';
 
-import { SizedIcon } from '@talend/design-system';
```

Comment:
Not sure to understand why you can't use design-system stuff in the storybook-docs.

Reply:
design-system depends on storybook-docs and with this code, storybook-docs depends on design-system = circular dep